src/applications/appeals/10182/tests/utils/issues.unit.spec.js

```diff
@@ -1,10 +1,12 @@
 import { expect } from 'chai';
-import moment from 'moment';
+import { subDays } from 'date-fns';
 import { SHOW_PART3 } from '../../constants';
 import { issuesNeedUpdating } from '../../utils/issues';
 
+import { parseDate } from '../../../shared/utils/dates';
+
 describe('issuesNeedUpdating', () => {
-  const yesterday = moment().format('YYYY-MM-DD');
+  const yesterday = parseDate(subDays(new Date(), 1));
```

Comment:
could this be `parseDateWithOffset({ days: -1 })`?

Reply:
Yes! I added `parseDateWithOffset` after this code and thought I went back and updated this pattern. Thanks!